domains/certificates/Query.API/API/Query.API/Startup.cs

```diff
@@ -13,7 +13,7 @@ public static void AddQueryApi(this IServiceCollection services)
     {
         services.ConfigureMarten(o =>
         {
-            o.Projections.Add<CertificatesByOwnerProjection>(ProjectionLifecycle.Inline);
+            o.Projections.Add(new CertificatesByOwnerProjection());
```

Comment:
What is the Lifecycle of the projection? It is not listed explicitly here, so is it Inline?

Reply:
It is inline